pandas/core/tools/datetimes.py

```diff
@@ -396,12 +400,16 @@ def _convert_listlike_datetimes(
             arg = arg.tz_convert(None).tz_localize("utc")
         return arg
 
-    elif is_datetime64_ns_dtype(arg_dtype):
+    elif is_datetime64_dtype(arg_dtype):
+        arg_dtype = cast(np.dtype, arg_dtype)
+        if not is_supported_unit(get_unit_from_dtype(arg_dtype)):
+            # We go to closest supported reso, i.e. "s"
+            arg = astype_overflowsafe(
+                np.asarray(arg), np.dtype("M8[s]"), coerce=errors == "coerce"
```

Comment:
How is errors="ignore" handled?

Reply:
will continue to incorrectly raise for the time being

Reply:
Gotcha. Planning on addressing in this PR or worth adding a TODO/xfail test?

Reply:
will add a TODO in my next "assorted" branch if this isn't a show-stopper

Reply:
(actually looks like this needs rebase so will add comment here)